deploy/helm/rawfile-localpv/templates/secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    {{- include "rawfile-localpv.labels" . | nindent 4 }}
+  name: {{ include "rawfile-localpv.fullname" . }}-secrets
+type: Opaque
+data:
+  internal-signature: {{ include "rawfile-localpv.internal-signature" . | b64enc }}
```

Comment:
Can we check if the secret already exists? 
Otherwise I suspect helm upgrade will trigger changes even if there are no changes